src/nm2/src/nm2_ipv6_address.c

```diff
@@ -113,6 +147,24 @@ void callback_IPv6_Address(
             return;
     }
 
+    /*
+     * (Askey) 
+     * Determine if the only change(s) was valid_lifetime and/or 
+     * preferred_lifetime, if so then skip nm2_ipv6_address_update() that 
+     * follows to avoid triggering the full-blown address provision and 
+     * disruption (can go as much as 6-10 seconds) on LAN side.
+     * To be aware, the skipping of nm2_ipv6_address_update() might come with 
+     * the side effect of not having correct stateful address lifetime on LAN 
+     * side, with the benefit of not causing LAN side disruption.
+     * 
+     * Should revisit once a better or complementary solution is in place.
+     */
+    if (mon->mon_type == OVSDB_UPDATE_MODIFY && !nm2_ipv6_address_is_modified(new))
+    {
+        LOG(INFO, "ipv6_addr: No need to modify IPv6 address.");
+        return;
+    }
```

Comment:
```suggestion
    if (mon->mon_type == OVSDB_UPDATE_MODIFY)
    {
        if (   new->enable_changed
            || new->address_status_changed
            || new->prefix_changed
            || new->address_changed
            || new->origin_changed
            || new->status_changed
           )
        {
            LOG(DEBUG, "ipv6_addr: No need to modify IPv6 address.");
            return;
        }
    }
```

Since the helper was only used here and it is not expected to be useful for anything else,
I propose to have this fix (especially since it might need to be revisited) in a single block.

Also note that log level was changed from INFO to DEBUG.